src/lib/holocene/day-of-week-picker.svelte

```diff
@@ -31,15 +31,23 @@
   <div class="flex gap-2 text-center">
     {#each genericWeekDays as { label, value }}
       {@const active = daysOfWeek.includes(value)}
-      <Button variant="secondary" {active} on:click={(e) => onClick(e, value)}>
+      <Button
+        variant="secondary"
+        class={active ? 'bg-interactive-secondary-active' : ''}
+        on:click={(e) => onClick(e, value)}
+      >
         {label}
       </Button>
     {/each}
   </div>
   <div class="flex flex-wrap gap-2 text-center">
     {#each weekDays as { label, value }}
       {@const active = daysOfWeek.includes(value)}
-      <Button variant="secondary" {active} on:click={(e) => onClick(e, value)}>
+      <Button
+        variant="secondary"
+        class={active ? 'bg-interactive-secondary-active' : ''}
```

Comment:
Could we do `active && 'bg-interactive-secondary-active'`?

Reply:
We totally could, [updated](https://github.com/temporalio/ui/pull/2118/commits/99186659a9e646c676efef78150e3eae50a37eda) ✅